paasta_tools/paasta_remote_run.py

```diff
@@ -415,6 +418,7 @@ def handle_interrupt(_signum, _frame):
             processor=processor,
             service=service,
             instance=instance,
+            role=native_job_config.get_role() or 'taskproc',
```

Comment:
can you make 'taskproc' a system_paasta_config entry for 'default_remote_run_role'?